src/demetriek/models.py

```diff
@@ -50,41 +56,48 @@ class Bluetooth(BaseModel):
     address: str
 
 
-class DisplayScreensaver(BaseModel):
+@dataclass
+class DisplayScreensaver(DataClassORJSONMixin):
     """Object holding the screensaver data of an LaMetric device."""
 
     enabled: bool
 
 
-class Display(BaseModel):
+@dataclass
+class Display(DataClassORJSONMixin):
     """Object holding the display state of an LaMetric device."""
 
     brightness: int
     brightness_mode: BrightnessMode
     width: int
     height: int
-    display_type: DisplayType | None = Field(default=None, alias="type")
     screensaver: DisplayScreensaver
+    display_type: DisplayType | None = field(
```

Comment:
We can avoid moving this stuff around by setting `kw_only=True` on the dataclasses.